src/steward.ts

```diff
@@ -71,7 +71,7 @@ export function handlePriceChange(event: PriceChange): void {
   V1.handlePriceChange(event);
 }
 export function handleForeclosure(event: Foreclosure): void {
-  // log.warning("handle foreclosure! {}", [event.block.hash.toHexString()]);
+ log.warning("[[[[[[[[[[[[SENDING FLOW TO handle foreclosure! {}", [event.block.hash.toHexString()]);
```

Comment:
I guess this is for your debugging?

BTW you can do new lines like this `\n`, eg:
```suggestion
 log.warning("\n\n\n TO handle foreclosure! {}\n\n\n", [event.block.hash.toHexString()]);
```
Puts 3 new lines above and below it.

Reply:
Awesome! will try this